dataset_conversion/convert_bids_to_nnUnetv2.py

```diff
@@ -10,9 +10,9 @@
 
 Usage example:
     python convert_bids_to_nnUNetv2.py --path-data ~/data/dataset --path-out ~/data/dataset-nnunet
-                    --dataset-name MyDataset --dataset-number 501 --split 0.6 0.2 0.2 --seed 99
+                    --dataset-name MyDataset --dataset-number 501 --split 0.6 0.2 --seed 99 --orient RPI
```

Comment:
I guess `--orient RPI` can be deleted since you dropped this idea.

```suggestion
                    --dataset-name MyDataset --dataset-number 501 --split 0.6 0.2 --seed 99
```